cmd/getStorageValue.go

```diff
@@ -0,0 +1,156 @@
+// Copyright © 2020 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/storage"
+	storage2 "github.com/makerdao/vulcanizedb/libraries/shared/storage"
+	"github.com/makerdao/vulcanizedb/libraries/shared/storage/types"
+	"github.com/makerdao/vulcanizedb/libraries/shared/transformer"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres/repositories"
+	"github.com/makerdao/vulcanizedb/utils"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var getStorageValueBlockNumber int64
+
+// getStorageValueCmd represents the getStorageValue command
+var getStorageValueCmd = &cobra.Command{
+	Use:   "getStorageValue",
+	Short: "Gets all storage values for configured contracts at the given block.",
+	Long: `Fetches and persists storage values of the configured contracts at a given block. It is important to note that the storage value gotten with this
+	command may not be different from the previous block in the database.`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		SubCommand = cmd.CalledAs()
+		LogWithCommand = *logrus.WithField("SubCommand", SubCommand)
+		fmt.Println("getStorageValue called")
```

Comment:
super nit-picky but maybe it makes sense to replace this with something like `fmt.Printf("getting storage values for all known keys at block %d", getStorageValueBlockNumber)`?